tests/test_static_views.py

```diff
@@ -298,6 +298,43 @@ def test_view_sandbox_topic(test_app_client, mock_rdu_user, stub_sandbox_topic_p
     assert page.h1.text.strip() == "Test sandbox topic page"
 
 
+def test_measure_page_social_sharing(
+    test_app_client, mock_rdu_user, stub_topic_page, stub_subtopic_page, stub_measure_page
+):
+    with test_app_client.session_transaction() as session:
+        session["user_id"] = mock_rdu_user.id
+
+    resp = test_app_client.get(
+        url_for(
+            "static_site.measure_page",
+            topic=stub_topic_page.uri,
+            subtopic=stub_subtopic_page.uri,
+            measure=stub_measure_page.uri,
+            version=stub_measure_page.version,
+        )
+    )
+
+    assert resp.status_code == 200
+    page = BeautifulSoup(resp.data.decode("utf-8"), "html.parser")
+
+    expected_social_attributes_and_values = {
+        "og:type": "article",
+        "og:title": "Test Measure Page",
+        "og:image": "https://www.ethnicity-facts-figures.service.gov.uk/static/images/opengraph-image.png",
```

Comment:
@samuelhwilliams we should inject the `RDU_SITE` env variable for the base URL here.

Reply:
Sure, done. 👍